posit_workbench_writing_and_reading_data.md

```diff
@@ -0,0 +1,56 @@
+# Optimum method for writing data to, and reading data from Stats
+
+## Background
+
+[Posit Team](https://posit.co/products/enterprise/team/) enterprise applications have been deployed for [Public Health Scotland (PHS)](https://publichealthscotland.scot/) on the [Microsoft Azure](https://azure.microsoft.com/en-gb/) cloud computing platform.
+
+## Purpose
+
+This document aims to provide users with information on the optimum method for writing data to, and reading data from Stats, when working in an R session in Posit Workbench.
+
+## Summary and Key Points
+
```

Comment:
Summary looks good, provides enough information on the '{arrow}' package and the benefits of using it for larger-than-memory datasets